typescript/src/client/evm/evm.types.ts

```diff
@@ -233,6 +234,20 @@ export interface RequestFaucetResult {
   transactionHash: Hex;
 }
 
+/**
+ * Options for sending an EVM transaction.
+ */
+export interface SendTransactionOptions {
+  /** The address of the account. */
+  address: Address;
+  /** The RLP-encoded transaction to sign and send, as a 0x-prefixed hex string. */
```

Comment:
Not necessarily RLP-encoded (if they provide a TransactionRequestEIP1559)

Reply:
Worth commenting here and on the "network" field that the chainId in the transaction is ignored